src/App.tsx

```diff
@@ -60,22 +60,35 @@ const PAGES = [
 const App = () => {
   const location = useLocation()
 
+  const [searchParams, setSearchParams] = useSearchParams()
+  const operatorId = searchParams.get('operatorId')
+  const lineNumber = searchParams.get('lineNumber')
+  const routeKey = searchParams.get('routeKey')
+  const timestamp = searchParams.get('timestamp')
```

Comment:
Maybe do apply this only for specific pages? Currently it will apply for all, also for pages that do not use these params (e.g. realtime map)

Reply:
You're right, this implementation is opinionated toward code simplicity. the URL may contain unnecessary artifacts in some cases. 
is it bad?

Reply:
Just was a bit weird when I switched between pages. If it is somehow possible to add another per-page boolean that will define wether to display the values or not, that would be great, but I'm not really familiar with the alternatives, so definitely whatever you think :)